AMD.Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.12.5-bookworm
```

Comment:
Would pytorch serve as a better base?
https://hub.docker.com/r/pytorch/pytorch/tags

Reply:
Probably yes, let me try. It might not require all the build packages after all.